src/renderer/drawers/OperationDetails/index.js

```diff
@@ -630,16 +630,19 @@ type OperationDetailsExtraProps = {
 };
 
 const OperationDetailsExtra = ({ extra }: OperationDetailsExtraProps) => {
-  return Object.entries(extra).map(([key, value]) => (
-    <OpDetailsSection key={key}>
-      <OpDetailsTitle>
-        <Trans i18nKey={`operationDetails.extra.${key}`} defaults={key} />
-      </OpDetailsTitle>
-      <OpDetailsData>
-        <Ellipsis>{value}</Ellipsis>
-      </OpDetailsData>
-    </OpDetailsSection>
-  ));
+  return Object.entries(extra).map(([key, value]) => {
+    if( typeof value == "object" || typeof value == "function") return null;
```

Comment:
Can you please explain this? In what case would value be an object or a function?

Reply:
I don't exactly remember now when this extra object can contain objects or functions, but I am sure I found that scenario while developing the integration. React failed because it cannot render objects.